template.yaml

```diff
@@ -33,7 +33,7 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       CodeUri:
-        Bucket: delete-expired-formstack-data-dist
+        Bucket: composer-dist
```

Comment:
We should remove this bucket name too. We can add a CFN Parameter and reference it:

```yaml
Parameters:
  DistributionBucket:
    Type: AWS::SSM::Parameter::Value<String>
    Default: /account/services/artifact.bucket
Resources:
    SubmissionDeletionLambda:
      Type: AWS::Serverless::Function
      Properties:
        CodeUri:
          Bucket: !Ref DistributionBucket
          Key: !Sub ${Stage}/delete-expired-formstack-data-lambdas/app.jar
```

Reply:
👍